content/en/blog/_posts/2023-12-14-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Cloud Provider Integrations Are Now Separate Components"
+date: 2023-12-14
```

Comment:
How about shifting this to later in the day?
```suggestion
date: 2023-12-14T09:30:00-08:00
```

Reply:
updated